app/services/location_service.rb

```diff
@@ -0,0 +1,21 @@
+class LocationService
+  class << self
+    def call_for_location(location, to_address)
+      response = conn.get("/directions/v2/route?to=#{to_address}&from=#{location}")
```

Comment:
It's more readable/maintainable if you break up your params in a block with a separate line for each param you set, similar to how you did it in `::conn`